### PR TITLE
Don't validate trailing whitespace for *.md files.

### PR DESCRIPTION
**Pull request by lukecwik**
Markdown file has certain lines that have trailing whitespace and they are legitimate as a way to prevent markdown rendering from collapsing the newlines.

This is a follow-up on #24951

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by lukecwik** [comment]
R: @kennknowles

**Comment by lukecwik** [comment]
Closed in favor of https://github.com/apache/beam/pull/25055, can always revert that and get this in.

File: sdks/python/test-suites/tox/py38/build.gradle [hunk 1]
```diff
@@ -120,7 +120,6 @@ task archiveFilesToLint(type: Zip) {
   destinationDirectory = file("$buildDir/dist")
 
   from ("$rootProject.projectDir") {
-    include "**/*.md"
```
[the remark this answers is not in the document]
Comment (kennknowles) [reply sdks/python/test-suites/tox/py38/build.gradle:123]: Empty line has a different effect. Line break tags are fine with me.

[the remark this answers is not in the document]
Comment (lukecwik) [reply sdks/python/test-suites/tox/py38/build.gradle:123]: I'm indifferent, just wanted to merge a solution to get the build back to green.

**Pull request closed without merging**